config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  get 'likes/likes'
-  get 'comments/comments'
-  resources :users, only: [:index, :show] do resources :posts, only: [:index, :show] end
+  resources :users, only: [:index, :show] do resources :posts, only: [:index, :show, :new] end
+
+    resources :posts, only: [:new, :create] do
+    resources :comments, only: [:create]
+    resources :likes, only: [:create]
```

Comment:
- [ ] Also add `new` route for comments.

Reply:
Done ✅